emc-metalnx-services/src/main/java/com/emc/metalnx/services/irods/CollectionServiceImpl.java

```diff
@@ -92,7 +92,7 @@ public class CollectionServiceImpl implements CollectionService {
 	private SpecificQueryProviderFactory specificQueryProviderFactory = new SpecificQueryProviderFactoryImpl();
 
 	@Autowired
-	private AdminServices adminServices;
+	private AdminServices userServices;
```

Comment:
What's the motivation behind renaming this variable?

Reply:
None.  Just missed when I changed it back to AdminServices.